jupyter_platform/cubeviz_leaflet_prototype_1.ipynb

```diff
@@ -35,7 +35,8 @@
     "\n",
     "import flask\n",
     "\n",
-    "%matplotlib inline\n",
+    "# NOTE: In-place plot update does not work with inline\n",
+    "%matplotlib notebook\n",
```

Comment:
@ibusko and @astrofrog , thanks for the tips. Turns out this was why it wasn't working for me. Works now with this change.